tests/test_actions.py

```diff
@@ -858,71 +850,57 @@ def test_kinking(self, Tester: Actor, mocker: MockerFixture) -> None:
         assert mock_flush.call_count == 1
 
     def test_skip_creation(self) -> None:
-        settings.UNABRIDGED_LOGGING = True
         fake_action = FakeAction()
+        mock_settings = ScreenPySettings(UNABRIDGED_NARRATION=True)
 
-        q = Silently(fake_action)
+        with mock.patch(self.settings_path, mock_settings):
+            q = Silently(fake_action)
 
         assert q is fake_action
 
-    def test_unabridge_from_function(self, Tester: Actor, mocker: MockerFixture) -> None:
-        settings.UNABRIDGED_LOGGING = True
+    def test_unabridge_from_function(
+        self, Tester: Actor, mocker: MockerFixture
+    ) -> None:
         mock_clear = mocker.spy(the_narrator, "clear_backup")
         mock_flush = mocker.spy(the_narrator, "flush_backup")
         mock_kink = mocker.spy(the_narrator, "mic_cable_kinked")
+        mock_settings = ScreenPySettings(UNABRIDGED_NARRATION=True)
 
-        Silently(FakeAction()).perform_as(Tester)
+        with mock.patch(self.settings_path, mock_settings):
+            Silently(FakeAction()).perform_as(Tester)
 
         assert mock_kink.call_count == 0
         assert mock_clear.call_count == 0
         assert mock_flush.call_count == 0
 
     def test_unabridged_from_class(self, Tester: Actor, mocker: MockerFixture) -> None:
-        settings.UNABRIDGED_LOGGING = True
         mock_clear = mocker.spy(the_narrator, "clear_backup")
         mock_flush = mocker.spy(the_narrator, "flush_backup")
         mock_kink = mocker.spy(the_narrator, "mic_cable_kinked")
+        mock_settings = ScreenPySettings(UNABRIDGED_NARRATION=True)
 
-        Tester.will(SilentlyPerformable(FakeAction()))
+        with mock.patch(self.settings_path, mock_settings):
+            Tester.will(SilentlyPerformable(FakeAction()))
 
         assert mock_kink.call_count == 1
         assert mock_clear.call_count == 1
         assert mock_flush.call_count == 1
 
-    def test_unabridged_set_inside_silently(self, Tester, caplog) -> None:
-        """
-        Confirm when unabridged flag is set, DEEP INSIDE actions which are
-        wrapped in Silently, that logging will occur normally.
-        """
-        caplog.set_level(logging.INFO)
-
-        Tester.will(Silently(Action1()))
-
-        assert [r.msg for r in caplog.records] == [
-            "Tester tries to Action1",
-            "    Tester tries to Action2",
-            "        Tester sees if simpleQuestion is equal to True.",
-            "            Tester examines SimpleQuestion",
-            "                => True",
-            "            ... hoping it's equal to True.",
-            "                => <True>",
-        ]
-
```

Comment:
I am not sure why this isn't included anymore.

Reply:
I believe i removed it when we decided `settings` should be immutable, since we couldn't do that anymore. I'll put it back now!